src/std/main.ab

```diff
@@ -239,3 +239,39 @@ pub fun get_env_var(var: Text): Text {
 pub fun load_env_file(): Null {
     unsafe $export "\$(xargs < .env)" > /dev/null$
 }
+
+pub fun shell_isset(name: Text): Bool {
+    $[[ ! -z \$\{!name+z} ]]$ failed {
+        return false
+    }
+    return true
+}
+
+pub fun shell_constant_set(name: Text, val: Text): Bool {
+    $readonly \$name="\$val" 2> /dev/null$ failed {
+        return false
+    }
+    return true
+}
+
+pub fun shell_constant_get(name: Text): Text {
```

Comment:
This function should also fail if the variable name provided is invalid

Reply:
I did the first tests but this didnt work related to #278

Reply:
@CymDeveloppement this has been fixed already in https://github.com/Ph0enixKM/Amber/pull/203

Reply:
We should release next version (amber `0.3.4-alpha` later today)

Reply:
Oh sorry!!